pkg/pillar/scripts/device-steps.sh

```diff
@@ -379,19 +375,23 @@ $BINDIR/diag -f -o /run/diag.pipe runAsService &
 
 echo "$(date -Ins -u) Starting tpmmgr as a service agent"
 $BINDIR/tpmmgr runAsService &
-wait_for_touch tpmmgr
-touch "$WATCHDOG_FILE/tpmmgr.touch"
```

Comment:
tpmmgr is not in AGENTS0, shouldn't be added in order to get .touch watchdog file created later?

Reply:
Nice catch, thank you. Hope it did not stopped in my experiments because of this issue.